Kudu.Client/Kudu.Client.csproj

```diff
@@ -7,6 +7,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Kudu.Client</RootNamespace>
     <AssemblyName>Kudu.Client</AssemblyName>
+    <NuGetPackageImportStamp>97469353</NuGetPackageImportStamp>
```

Comment:
thank you. will remove. didn`t look into this earlier, thought is an VS thing, since we have this tag in other project as well. I will do a clean up after merge this change.

```
E:\kudu>findstr /sinp /c:"NuGetPackageImportStamp" *.csproj
Kudu.Client\Kudu.Client.csproj:10:    <NuGetPackageImportStamp>97469353</NuGetPackageImportStamp>
Kudu.Console\Kudu.Console.csproj:12:    <NuGetPackageImportStamp>5bb80beb</NuGetPackageImportStamp>
Kudu.Contracts\Kudu.Contracts.csproj:10:    <NuGetPackageImportStamp>c86185d4</NuGetPackageImportStamp>
Kudu.Core\Kudu.Core.csproj:11:    <NuGetPackageImportStamp>4ba3ba66</NuGetPackageImportStamp>
Kudu.Services\Kudu.Services.csproj:12:    <NuGetPackageImportStamp>4c6b70dd</NuGetPackageImportStamp>
Kudu.Services.Web\Kudu.Services.Web.csproj:18:    <NuGetPackageImportStamp>bf926998</NuGetPackageImportStamp>
```